test/e2e/app-dir/next-after/app/api/dynamic/edge/route.ts

```diff
@@ -0,0 +1,14 @@
+import { NextRequest, NextResponse } from 'next/server'
+import { after } from 'next/server'
+
+export const runtime = 'edge'
+
+export async function GET(request: NextRequest): Promise<NextResponse> {
+  after(async () => {
+    console.log('after api/dynamic/edge')
+  })
+  return NextResponse.json({
```

Comment:
Can we make these tests more complicated to ensure that `after` is called after the response is fully complete, including body?